volatility3/framework/plugins/windows/verinfo.py

```diff
@@ -1,204 +0,0 @@
-# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Mmmmm, not too keen on this bit...  5;P

Reply:
whoops, seems I wanted to drop the change made to this file by my branch and accidentally deleted the entire file :D